TheThingsNetwork/lorawan-stack
# Fix websocket stream reconnection

#### Summary
This PR fixes the automatic reconnection of event streams after a disruption.

#### Changes

- Fix omitted connection success event to setup the internal state machine correctly for reconnection attempts.
- Improve the synthetic event messages to reflect the new situation for disconnects after the websocket refactor.

#### Testing
- Go to the Console and navigate to an application or gateway
- Observe the event stream connecting and delivering events
- Put your machine into sleep mode
- Wait a couple of seconds
- Exit sleep mode
- Observe the event stream being closed
- Observe the event stream being automatically reconnected after a couple of seconds

#### Notes for Reviewers
`START_EVENT_STREAM_SUCCESS` event and hence the connection was never set as `CONNECTED` but `CONNECTING` and consequently no reconnection attempts were made because the connection was considered to be still connecting.

#### Checklist
